persistence/src/main/scala/zio/actors/persistence/PersistenceConfig.scala

```diff
@@ -0,0 +1,58 @@
+package zio.actors.persistence
+
+import zio.{ DefaultRuntime, IO, Managed, Promise, Task, UIO }
+import zio.config.ConfigDescriptor
+import zio.config.ConfigDescriptor._
+import zio.actors.ActorsConfig._
+
+private[actors] object PersistenceConfig {
+
+  private lazy val runtime = new DefaultRuntime {}
```

Comment:
Why?

Reply:
It's the same reason as two comments above about same runtime in `JDBCJournal`, so I will copy-paste it a bit:

I think at the moment that this is required by the nature of ActorSystem.

We need to load configuration file to map plugin definitions with class names that will be reflected. And ideally we would like loading to happen only once - no need to load file separately by every persistent actor. 

To achieve it I decided to use lazy val runtime.unsafeRun so when actors arrive they already find a configuration already loaded that can be shared.

Thanks to this this config won't be loaded each time from config file but only once by the first actor in ActorSystem.

What do you think about it?

Reply:
Could you just sketch the behavior you want to avoid? You're probably right, but I'm failing to track the entire reasoning behind. :pray:

Reply:
Sure, here you go:
![diagram](https://i.ibb.co/KFHb7GX/diagram.png)

So as only Persistent Actors use Persistence Module it would require to access config file for datastore bindings each actor is created. So introducing a `Promise` that will be already there by `runtime.unsafeRun...` actors can use config that was loaded from file only once. I think this skew is caused by actor model foundations, as `ActorSystem` don't know of concrete Statefuls, persistence ect.
WDYT?

Reply:
Hmm I assumed that we don't want those class names in code directly but via some config file, is that right?